dashboard/app/helpers/levels_helper.rb

```diff
@@ -384,4 +384,14 @@ def enable_scrolling?
   def enable_examples?
     current_user && current_user.admin? && @level.is_a?(Blockly)
   end
+
+  # If this is a restricted level (i.e. applab) and user is under 13, redirect with a flash alert
+  def redirect_under_13(level)
+    return unless level.game == Game.applab
+
+    if current_user && current_user.age && current_user.under_13?
```

Comment:
we usually treat age.nil? as < 13 (this is already baked in to the under_13? method).

This should not be common because we now require age on sign in and have an interstitial "forcing" users who created an account before this requirement to add an age.